collections/sentinel-3-l1b-olci.yaml

```diff
@@ -102,12 +102,14 @@ CubeDimensions:
     extent:
       - -180
       - 180
+    reference_system: 4326
   y:
     type: spatial
     axis: y
     extent:
       - -85
       - 85
+    reference_system: 4326
   t:
     type: temporal
     extent:
```

Comment:
Revisit time according to docs:
<html>
<body>
<!--StartFragment-->

Revisit time | < 2 days with 2 satellites
-- | --


<!--EndFragment-->
</body>
</html>